lte/gateway/c/core/oai/tasks/nas/ies/ApnAggregateMaximumBitRate.c

```diff
@@ -112,53 +112,121 @@ int encode_apn_aggregate_maximum_bit_rate(
 void bit_rate_value_to_eps_qos(
     ApnAggregateMaximumBitRate* apn_ambr, uint64_t ambr_dl, uint64_t ambr_ul) {
   uint64_t ambr_dl_kbps = ambr_dl / 1000;  // ambr_dl is expected in bps
-  uint64_t ambr_ul_kbps = ambr_ul / 1000;  // ambr_ul is expected in bps
-  if (ambr_dl_kbps == 0) {
+  uint64_t ambr_ul_kbps = ambr_ul / 1000;  // ambr_ul is expected in bps 8640000
```

Comment:
what is the relevance of `8640000` in this comment?

Reply:
Updated.